insurance-service/azure-pipelines.yml

```
@@ -0,0 +1,229 @@
+  condition: succeeded()
+  variables:
+    appVersion: $[ stageDependencies.Build.BuildAndTest.outputs['SetVersion.appVersion'] ]
+  jobs:
+    - deployment: Deployment_Staging
+      environment:
+        name: staging
+      strategy:
+        runOnce:
+          deploy:
+            steps:
+              - checkout: git://gitops
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)